osbrain/tests/test_agent_ipc_sockets.py

```diff
@@ -9,6 +10,9 @@
 
 from common import nsproxy  # pragma: no flakes
 
+pytestmark = pytest.mark.skipif(sys.platform == 'win32',
+                                reason='IPC not available on Windows')
```

Comment:
Does `pytestmark = skip_windows_ipc` work? (importing the corresponding `skip_windows_ipc` definition).

This way we put all `pytest.mark.skipif` declarations together.